CI/setup_cvmfs_lcg105.sh

```diff
@@ -0,0 +1,23 @@
+# setup appropriate LCG release via cvmfs
+
+if test -e /etc/centos-release && grep 'CentOS Linux release 7' /etc/centos-release; then
+  lcg_os=centos7
+elif test -e /etc/centos-release && grep 'CentOS Stream release 8' /etc/centos-release; then
+  lcg_os=centos8
+# not centos. Check for RHEL  
+elif test -e /etc/redhat-release && grep 'Red Hat Enterprise Linux release 9' /etc/redhat-release; then
```

Comment:
```suggestion
elif test -e /etc/redhat-release && grep 'Linux release 9' /etc/redhat-release; then
```
This would allow this to work also on Alma 9 (used widely at CERN) and Rocky 9 (used elsewhere, eg. at RAL). All of them can use `lcg_os=el9`.

For the record, here is the content of `/etc/redhat-release` for a few OS installations:
```
Red Hat Enterprise Linux release 9.3 (Plow)
AlmaLinux release 9.3 (Shamrock Pampas Cat)
Rocky Linux release 9.3 (Blue Onyx)
```

Reply:
Thank you @timadye . I've committed your suggestion